packages/main/src/BusyIndicator.hbs

```diff
@@ -1,12 +1,18 @@
 <div class="ui5-busyindicator-root">
-
-	{{#if active}}
-		<div class="ui5-busyindicator-dynamic-content" tabindex="0" role="progressbar" aria-valuemin="0" aria-valuemax="100" title="{{ariaTitle}}">
-			<div class="ui5-busyindicator-circle circle-animation-0"></div>
-			<div class="ui5-busyindicator-circle circle-animation-1"></div>
-			<div class="ui5-busyindicator-circle circle-animation-2"></div>
-		</div>
-	{{/if}}
+	<div class="ui5-busyindicator-wrapper">
+		{{#if active}}
+			<div class="ui5-busyindicator-dynamic-content" tabindex="0" role="progressbar" aria-valuemin="0" aria-valuemax="100" title="{{ariaTitle}}">
+				<div class="ui5-busyindicator-circle circle-animation-0"></div>
+				<div class="ui5-busyindicator-circle circle-animation-1"></div>
+				<div class="ui5-busyindicator-circle circle-animation-2"></div>
+			</div>
+		{{/if}}
+		{{#if text.length}}
+			<ui5-label class="ui5-busyindicator-text">
```

Comment:
Can you sync it with the VD (I think A. Kirichev is responsible), because there is no design for text in FIori 3 VD wiki, although there is such property in openui5. Maybe they should update their design or I don't know.

Reply:
Synched with him and added some additional space